cmd/user-auth/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/db"
+	"github.com/edulinq/autograder/internal/exit"
+)
+
+func TestMain(suite *testing.M) {
+	// Run inside a func so defers will run before exit.Exit().
+	code := func() int {
+		db.PrepForTestingMain()
+		defer db.CleanupTestingMain()
+
+		return suite.Run()
+	}()
+
+	exit.Exit(code)
+}
+
+// This test ensures a CMD can start its own server.
+// Since this is the only test that ensures a CMD can start its own server,
+// it must remain or be replaced with an equivalent test if removed.
+func TestUserAuthBase(test *testing.T) {
+	// CMDs always succeed in user authentication, regardless of credentials, so only one test case is needed.
+	testCase := struct {
+		cmd.CommonCMDTestCase
+		targetEmail string
+		targetPass  string
+	}{
```

Comment:
Why have this struct if there is only one test case?
This struct only makes sense if you are looping over them.